brdocument/cpf.go

```diff
@@ -0,0 +1,42 @@
+package brdocument
+
+import (
+	"regexp"
+	"fmt"
+)
+
+const (
+	LENGTH = 11
+	LABEL = "CPF"
+	REGEX = `^([\d]{3})([\d]{3})([\d]{3})([\d]{2})$`
+	NUMBER_OF_DIGITS = 2
+)
```

Comment:
when add other document it will conflict with other constant, so I suggest type it as a Cpf